packages/recomponents/src/components/r-tabs/r-tab.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-    <div v-show='isActive'>
+    <div :id="tabPanelId"
+         role="tabpanel"
+         :aria-labelledby="tabId"
+         v-if="$slots.default"
```

Comment:
I'm concerned about the usage of `v-if` here: does the bug below manifest at all?
https://jira.ilabsinc.com/browse/GWY-13918

Reply:
Hmm.. the bug didn't manifest at all 🤔 
I moved the contents of `v-show` to `v-if` and tested:
* In Storybook, but with [this template](https://github.com/Rebilly/rebilly-recomponents/blob/f5c712a8ce12ba86762a312e2e731b3fc2ec00c5/packages/recomponents/src/components/r-tabs/r-tabs.story.js#L8-L24) which is different than the one currently in `master`
* In Recomm, where I mocked Recomponents with a local build

Everything works fine and by using only `v-if` we are able lazy load tab content, just like you suggested in the ticket 🎉 

Looks like the bug might be was caused by `RTile` and/or it's name slots?

Reply:
Hmm. Interesting that it doesn't show up! But thank you for being so thorough in testing it. Cheers!